yorkie/src/test/kotlin/dev/yorkie/util/SplayTreeSetTest.kt

```diff
@@ -0,0 +1,114 @@
+package dev.yorkie.util
+
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class SplayTreeSetTest {
+    private lateinit var target: SplayTreeSet<String>
+
+    @Before
+    fun setUp() {
+        target = SplayTreeSet { it.length }
+    }
+
+    @Test
+    fun `should conform to specs when insertions and splaying occur`() {
+        assertEquals(0, target.length)
+
+        target.insert("A2")
+        assertEquals("[2,2]A2", target.getStructureAsString())
```

Comment:
I think `getStructureAsString` is intuitive and good. 👍 
It would be good to change other SDKs as well.